sct.py

```diff
@@ -1379,7 +1379,7 @@ def create_test_release_jobs(branch, username, password, sct_branch, sct_repo):
                               sct_branch_name=sct_branch, sct_repo=sct_repo)
     base_path = f'{server.base_sct_dir}/jenkins-pipelines'
 
-    enterprise_to_filter_out = r'ldap|ics|-sla|saslauthd'
+    enterprise_to_filter_out = r'ldap|ics|-sla|saslauthd|fips'
```

Comment:
if it's filtered here, where exactly it's gonna be generated ?
i.e. there no enterprise only artifacts jobs till now

Reply:
well, if it will not be created, we will need to add in the enterprise code to add also artifacts (like we have for upgrade, and longevity jobs)